locale/fr/foreman_tasks.po

```diff
@@ -1,667 +1,665 @@
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# French translations for foreman_tasks package.
+# Copyright (C) 2023 THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the foreman_tasks package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
 #
-# Translators:
-# Baptiste Agasse <baptiste.agasse@gmail.com>, 2018-2019
-# Claer <transiblu@claer.hammock.fr>, 2016
 msgid ""
 msgstr ""
-"Project-Id-Version: foreman_tasks 8.0.1\n"
+"Project-Id-Version: foreman_tasks 1.0.0\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2016-02-12 14:04+0000\n"
-"Last-Translator: Claer <transiblu@claer.hammock.fr>, 2016\n"
-"Language-Team: French (http://app.transifex.com/foreman/foreman/language/fr/)\n"
+"PO-Revision-Date: 2023-11-28 14:12+0100\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: French\n"
+"Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Language: fr\n"
-"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 100000"
-"0 == 0 ? 1 : 2;\n"
+"Plural-Forms: nplurals=; plural=;\n"
+"\n"
 
 msgid "%(last)s Last %(time)s"
-msgstr "%(last)s Dernier %(time)s"
+msgstr ""
```

Comment:
This is wrong - it's not pulling in any translations and actually removing them.